packager/mpd/base/mpd_builder.h

```diff
@@ -32,6 +32,15 @@ class AdaptationSet;
 class MediaInfo;
 class Period;
 
+class Clock {
```

Comment:
I had to create something similar already in Muxer in b1095f6bc36f38892c848689135b3d1dab1e7e1b.  See packager/media/base/muxer.h.

We can always create a new folder to hold these things if we want to dedup them.  I agree that this doesn't really belong in mpd.

Reply:
moved it to a separate file. This is used in the end to print ISO-8601 dates so I think it's better to keep this in std::tm instead of uint64_t timestamps. I updated the MuxerClock to use this Clock abstraction as well.